src/ui/shared/CountryFlag/CountryFlag.module.scss

```
@@ -0,0 +1,17 @@
+.flag {
+  width: ws(18);
+  height: ws(13);
+  border-radius: 1px;
+  // The sprite is a background image, so the package's `:before` spacer would
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)